dev/lib/product_taxonomy/commands/command.rb

```diff
@@ -20,9 +20,9 @@ def run(...)
         execute(...)
       end
       logger.info("Completed in #{elapsed.round(2)} seconds")
-    rescue => e
-      logger.error("\e[1;31mError:\e[0m #{e.message}")
-      exit(1)
+    # rescue => e
+      # logger.error("\e[1;31mError:\e[0m #{e.message}")
+      # exit(1)
```

Comment:
Should we delete these lines?

Reply:
Yikes, didn't mean to include that change. Reverted.